event-log-verification-service/src/main/scala/com/ubirch/verification/Service.scala

```diff
@@ -3,11 +3,18 @@ package com.ubirch.verification
 import com.ubirch.util.Boot
 import com.ubirch.verification.util.udash.JettyServer
 
+import scala.util.{Failure, Success}
+
 object Service extends Boot(LookupServiceBinder.modules) {
 
   def main(args: Array[String]): Unit = {
 
-    get[JettyServer].startAndJoin()
+    getAsTry[JettyServer] match {
+      case Failure(e) =>
+        logger.error("Error at system initialization", e.getMessage, e)
```

Comment:
is the string interpolation for the "e.getMessage" missing?

Reply:
Changed in [ec8d998](https://github.com/ubirch/ubirch-event-log/pull/92/commits/ec8d99815a81d58215abca8931ca978041688f61)